src/include/queries.h

```diff
@@ -86,6 +86,17 @@ pgexporter_query_get_functions(int server, struct query** query);
 int
 pgexporter_query_execute(int server, char* sql, char* tag, struct query** query);
 
+/**
+
+ * Extracts the server's version as a char (minimum integral data type), and sets
+ * the value of `config->servers[server].version` to it.
+ *
+ * @param server The server index in config
+ * @return Server version as a char data type
+ */
```

Comment:
We are not really doing that though...

Thinking about it - lets just keep it as a status code - so `int`, and assign internally in the function

Reply:
Oh I am really sorry about this. I was travelling back to my place when I made this change and pushed as soon as I got internet, and it was a lapse of judgement on my part.

Yes I am indeed assigning the version internally and the returned value is just 0 if all's ok and 1 if not.

I'll change it back immediately.

Reply:
np